scripts/versionDocFolder.js

```diff
@@ -1,4 +1,4 @@
-const {parse} = require('semver');
+const regexMajorMinor = /^(\d+)\.(\d+)/;
```

Comment:
had a look quickly to the library, shouldn't you use `validateAndParse`? https://github.com/omichelsen/compare-versions/blob/main/src/utils.ts#L9C14-L9C30

Reply:
my bad, they do not export utils